Bottom: 753e29639fbb21ba0934e409dbbc5a3dbbb8df15
Top:    901a0ede6026559d22af1d33bbbe98698e8adf6f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-09 17:07:06 -0500

Add parameter for buildbot version



---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 521c823..dde9cdb 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -1,4 +1,5 @@
-FROM        buildbot/buildbot-master:v0.9.13
+ARG         BUILDBOT_VERSION=latest
+FROM        buildbot/buildbot-master:$BUILDBOT_VERSION
 MAINTAINER  Andrew Cooper <andrew.cooper@novatechweb.com>
 
 WORKDIR /buildbot
